src/components/bixicard.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import LocalizedStrings from 'react-localization';
+
+let strings = new LocalizedStrings({
+    en: {
+        'theres': 'There are',
+        'bikes': ' Bixi bicycle',
+        'available': ' available at Milton/Clark'
+    },
+    fr: {
+        'theres': 'Il y a',
+        'bikes': 'vélo',
+        'available': ' Bixi disponible à Milton/Clark'
+    }
+});
+
+export default class BixiCard extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            //data: [],
+            ba: 0,
+        };
+        this.updateBixi = this.updateBixi.bind(this);
+    }
+
+    updateBixi() {
+        fetch('https://secure.bixi.com/data/stations.json')
+        .then(res => res.json())
+        .then(res => {
+            //this.setState({data: res.stations});
+            res.stations.filter(function(station){
+                // milton/clark station 6209
+                return station.n === "6209";
+            }).map(function(station){
+                // set state of bikes available
+                this.setState({ba: station.ba});
+               // console.log('Bixi at', Date.now(), this.state.ba)
+            }, this);
+        }).catch(err => {
+            console.log('Bixi error '+err);
```

Comment:
Maybe change the handling to an online display? Maybe something like 'Unable to fetch Bixi availability'?